### PR TITLE
Refactor styles

### PR DESCRIPTION
**Pull request by davidyuk**
fixes partially #740

### CODE REVIEW

File: src/components/tipRecords/TipMedia.vue [hunk 1]
```diff
@@ -1,106 +1,91 @@
...
+      });
+    },
+  },
 };
 </script>
 
 <style lang="scss" scoped>
 .tip-media {
-  padding-top: 55%;
+  max-height: 210px;
```
Comment (davidyuk) [inline src/components/tipRecords/TipMedia.vue:42]: @AtanasKrondev how height should be actually defined? I can't get it from the code

Comment (AtanasKrondev) [reply src/components/tipRecords/TipMedia.vue:42]: According to the design it should be 16:9 ratio.

Comment (AtanasKrondev) [reply src/components/tipRecords/TipMedia.vue:42]:
```suggestion
  aspect-ratio: 16/9;

  @supports not (aspect-ratio: 16/9) {
    max-height: 360px;
    @include smallest {
      max-height: 210px;
    }
  }
```

<a href="https://caniuse.com/mdn-css_properties_aspect-ratio" target="_blank">Compatibility</a>. Honestly I didn't know about the ```aspect-ratio``` property and I don't vouch for it but seems suitable in this case.🤷🏻‍♂️

**Review by Liubov-crypto — COMMENTED** [review]
Tested in Safari, FF, Chrome browsers with extension and iframe wallet, LGTM:
Trending - pass 
Search - pass
Change profile/back pics - pass 
Change profile info - pass 
Cookies - pass 
Tip on the Feed - pass 
Retip on the feed - pass  
Post without tipping - pass 
Comment - pass 
Tip comment on the feed - pass 
Pin/Unpin - pass 
Claim - pass 
Report post - pass 
Voting - pass 
Vote - pass 
Revoke Vote - pass 
Delegate voting power - pass 
Create a new poll - pass 
Search - pass 
Meet - pass 
Updates - pass 
Faq- pass 

and bugs remain #1111 #1112 (but maybe this is the side of the wallet)

File: src/components/tipRecords/TipMedia.vue (hunk 1 above)
Comment (davidyuk) [reply src/components/tipRecords/TipMedia.vue]: also in the failback, we can define the proper height for each breakpoint, check the last commit

**Review by AtanasKrondev — APPROVED** [review]

**Pull request merged**